dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/task/FormatTask.java

```diff
@@ -39,7 +39,7 @@ protected Void doRun(TaskRunContext context, ByteSink sink, Handle handle) throw
   }
 
   @Override
-  public String toString() {
-    return "Write " + getTargetPath() + " containing '" + format + "'.";
+  public String describeSourceData() {
+    return "containing '" + format + "'.";
```

Comment:
nit, optional: Consider using String.format() instead to avoid things like " " and "'" and "'."

Reply:
(And similarly for other describeSourceData() overrides.)

Reply:
Done.